api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -503,19 +504,21 @@ public Workspace setPublished(
   }
 
   @Override
-  public UserRecentWorkspace updateRecentWorkspaces(long workspaceId, long userId, Timestamp lastAccessDate) {
+  public UserRecentWorkspace updateRecentWorkspaces(long workspaceId, long userId) {
```

Comment:
One thing to think about, which may not matter, is that if you don't pass in a timestamp here, the access times recorded somewhere else will probably not match what's put in the table exactly. For example, if the workspace had a last accessed time (regardless of user), it might be off by a few milliseconds from what's here. Probably ok for this application, but it made me scratch my head for a minute.